### PR TITLE
We should always show permanent actions

### LINKED ISSUE #2407: Trashbin: no "restore" action and missing previews
The trash bin no longer shows a undelete operation.

trash

Also there is no preview if encryption is enabled. Both works on stable10:

trash - stable10

cc @rullzer for the previews

Regarding the "restore" action, the code is still in the template, don't know why it is no longer visible https://github.com/nextcloud/server/blob/master/apps/files_trashbin/templates/index.php#L34

### PR DESCRIPTION
**Pull request by rullzer**
Should fix #2407

But it is CSS magic so summoning @nextcloud/designers 

@schiessle @nickvergessen please test.

### CODE REVIEW

**Comment by skjnldsv** [comment]
👍

**Comment by rullzer** [comment]
According to my bisect the commit that broke it is b75ef0f8b3dea718b52b0dedcac16b504374b7cd

so cc @skjnldsv

**Comment by schiessle** [comment]
great, this brings the restore button back! There is one small glitch, the mouse pointer changes above and below the "restore" button to indicate that you can click on it but not if you point directly to the icon or the "restore" label.

**Comment by rullzer** [comment]
Somebody that actually knows CSS and the like. Please take over ;)

**Comment by schiessle** [comment]
fixed the mouse pointer stuff... 👍

**Comment by rullzer** [comment]
Fine by me!

**Comment by MorrisJobke** [comment]
Thanks - I added this to bring back the darker color for shared entries. This still works, so this is 👍  also from me :)

File: apps/files/css/files.css [hunk 1]
```diff
@@ -662,20 +662,16 @@ html.ie8 .column-mtime .selectedActions {
...
 /* show share action of shared items darker to distinguish from non-shared */
 #fileList a.action.action-share.permanent.shared-style,
-#fileList a.action.action-favorite.permanent,
 /* show hovered permanent entries darker */
-#fileList tr a.action.action-share.permanent:hover,
-#fileList tr a.action.action-share.permanent:focus,
-#fileList tr a.action.action-menu.permanent:hover,
-#fileList tr a.action.action-menu.permanent:focus {
+#fileList tr a.action.action.permanent:hover,
+#fileList tr a.action.action.permanent:focus {
```
Comment (jancborchardt) [inline apps/files/css/files.css]: The removal of this specificity causes the menu entries being way too big. Can you add that back @rullzer?
capture du 2016-12-02 14-41-33
cc @nextcloud/designers

**Pull request merged**